lib/Remote/RemoteWebElement.php

```diff
@@ -477,6 +485,39 @@ public function equals(WebDriverElement $other)
         ]);
     }
 
+    /**
+     * Attempt to click on a child level element.
+     *
+     * This provides a workaround for geckodriver bug 653 whereby a link whose first element is a block-level element
+     * throws an ElementNotInteractableException could not scroll into view exception.
+     *
+     * The workaround provided here attempts to click on a child node of the element.
+     * In case the first child is hidden, other elements are processed until we run out of elements.
+     *
+     * @param ElementNotInteractableException $originalException The exception to throw if unable to click on any child
+     * @return RemoteWebElement the current instance.
```

Comment:
I think there is no reason to provide fluent interface. This could just be kept as void.